packages/docs/components/page-configs/contribution/documentation-page/page-config.ts

```diff
@@ -0,0 +1,140 @@
+import { BlockType, ApiDocsBlock } from '../../../../types/configTypes'
+import blocktypes from './block-types'
+import apiOptions from './api-options'
+export default [
+  {
+    type: BlockType.TITLE,
+    translationString: 'documentationPage.title',
+  },
+  {
+    type: BlockType.PARAGRAPH,
+    translationString: 'documentationPage.description',
+  },
+  {
+    type: BlockType.SUBTITLE,
+    translationString: 'documentationPage.pageConfigTitle',
+  },
+  {
+    type: BlockType.PARAGRAPH,
+    translationString: 'documentationPage.pageConfigText',
+  },
+  // title
+  {
+    type: BlockType.HEADLINE,
+    translationString: 'documentationPage.blocktypes.title.title',
+  },
+  {
+    type: BlockType.PARAGRAPH,
+    translationString: 'documentationPage.blocktypes.title.text',
+  },
+  {
+    type: BlockType.CODE,
+    code: blocktypes.title,
+  },
+  // subtitle
+  {
+    type: BlockType.HEADLINE,
+    translationString: 'documentationPage.blocktypes.subtitle.title',
+  },
+  {
+    type: BlockType.PARAGRAPH,
+    translationString: 'documentationPage.blocktypes.subtitle.text',
+  },
+  {
+    type: BlockType.CODE,
+    code: blocktypes.subtitle,
+  },
```

Comment:
did you try to add blocktypes.subtitle directly below the example? I.e. you'll be able to see what it compiles to.